.npmrc

```
@@ -0,0 +1,3 @@
+registry=https://msdata.pkgs.visualstudio.com/SQLToolsAndLibraries/_packaging/SQLToolsAndLibraries_PublicPackages/npm/registry/
```

This needs to be pointing to the public registry, NOT our private feed (we shouldn't have anything about our private feeds in this repo)

```suggestion
registry=https://registry.npmjs.org
```

---

It's fine (and expected) for our PR build to use the public registries. It's only the ADO pipelines that need to use the private stuff.